en-US/win10/samples/DriverLab4.md

```diff
@@ -45,14 +44,13 @@ Your target device is now configured for driver development using Visual Studio.
 * Click Next to move to the next step.
 * Assure that Host IP matches the IP address of your development machine.
 * Click Next to move to the next step.
-* Click Finish to complete the configuration.
-	
- Configure your Visual Studio project 
+* Click Finish to complete the configuration.   
+* Configure your Visual Studio project 
```

Comment:
Shouldn't this line NOT be a bullet point? Isn't it a sub-header to group the bullet points that follow it? On its own, its not descriptive